float32 latitude
float32 longitude
float32 altitude
float32 yaw
float32 roll
float32 pitch